cirq-core/cirq/work/observable_measurement_test.py

```diff
@@ -543,3 +544,29 @@ def test_measure_observable_bad_grouper():
             stopping_criteria=RepetitionsStoppingCriteria(50_000),
             grouper='super fancy grouper',
         )
+
+
+@pytest.mark.slow
```

Comment:
On my side `check/pytest -n 0 cirq-core/cirq/work/observable_measurement_test.py` produces the following warning -

> PytestUnknownMarkWarning: Unknown pytest.mark.slow - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/how-to/mark.html

Please define the mark or remove it.